circleguard/utils.py

```diff
@@ -120,7 +121,10 @@ def __init__(self, replay, cursor_color):
         self.cursor_color = cursor_color
         self.username = replay.username
         self.t = replay.t
-        self.xy = replay.xy
+        # copy so we don't flip the actual replay's xy coordinates when we
+        # account for hr (not doing this causes replays to be flipped on odd
+        # runs of the visualizer and correct on even runs of the visualizer)
+        self.xy = np.copy(replay.xy)
```

Comment:
just do `replay.xy.copy()`, does the same without the numpy import

Reply:
isn't that using the `list.copy` method? There must be a reason why numpy is providing their own copy method... @samuelhklumpers any insight?

Reply:
no, it's https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.copy.html

Reply:
I see - I don't know anything about how memory layout affects us 

> Note that this function and numpy.copy are very similar, but have different default values for their order= arguments.

but that does seem better otherwise

Reply:
memory layout should not concern us unless we wanted to attempt to lay out our arrays so that we optimize the reading order for most operations. For a less than marginal performance gain, probably.